cygnus-ngsi/spec/SPECS/cygnus-ngsi.spec

```diff
@@ -99,6 +99,15 @@ rm -rf $RPM_BUILD_ROOT
 %{_project_install_dir}
 
 %changelog
+* Fri Sep 4 2020 Alvaro Vega <alvaro.vegagarcia@telefonica.com> 2.4.0
+- [cygnus-ngsi][RowAggregator] Handle metadata as json. This is to upgrade json-like sinks behaviour. (#1902)
+- [cygnus-ngsi][ColumnAggregator] Fix attribute type key to make it unique. (#1904)
+- [cygnus-ngsi][PostgisSink, PostgreSQLSink] Implement records expiration for Postgis and PostgreSQL sinks (#1915)
+- [cygnus-ngsi][ColumnAggregator] Handle metadata as json. This is to upgrade json-like sinks behaviour. (#1902)
+- [cygnus-ngsi][CKANSink] Fix bug about bad service name used by CKAN orgName with a name mapping (#1910)
+- [cygnus-ngsi][CKANSink] Add flag option to disable metatada in CKAN (#1912)
+- [cygnus-common][SQLBackendImpl] Exception handle for connection/statement SQL objects (#1907)
```

Comment:
This line should me moved to cygnus-commons .spec

Reply:
That's right! Fixed in 9d167ab5276e8dc6f157493045a073576a87d92b